### ISSUE TITLE
Markdown version of API specs

### ISSUE THREAD

**Issue by arun-gupta** [opea-project/docs#103]
The current API specs are available as a binary document at https://github.com/opea-project/docs/blob/main/OPEA_API_Specification_v0.9.docx. These documents are hard to collaborate and consume.

Please provide the normative version as markdown. DOCX can be a supporting document but the definitive document should be text-based.

**Comment by dbkinder** [comment]
Already in progress.  See PR #94 
Alas this API content is manually created and maintained where it should be created programmatically.

**Comment by arun-gupta** [comment]
We should deprecate manual creation of the doc and only publish the programmatic one, in a text format.

**Comment by dbkinder** [comment]
> We should deprecate manual creation of the doc and only publish the programmatic one, in a text format.

I completely agree.  How do we get the programmatic JSON description of the API?

**Comment by arun-gupta** [comment]
That would be an engineering team question. @kding1 would know

**Comment by ftian1** [comment]
> > We should deprecate manual creation of the doc and only publish the programmatic one, in a text format.
> 
> I completely agree. How do we get the programmatic JSON description of the API?

we will work on that

**Comment by yinghu5** [comment]
Hi @ftian1 just remind and please update if news.  thank you!
